macos/FluentUI/Button/Button.swift

```diff
@@ -5,158 +5,254 @@
+	@objc public convenience init(
+		title: String,
+		size: ButtonSize = .large,
+		style: ButtonStyle = .primary,
+		accentColor: NSColor = Colors.primary
+	) {
+		let format = ButtonFormat(size: size, style: style, accentColor: accentColor)
+		self.init(title: title, format: format)
 	}
 
-	/// Initializes a Fluent UI Button with an image, setting the imagePosition to imageOnly
+	/// Initializes a Fluent UI Button with an image only
 	/// - Parameters:
 	///   - image: The NSImage to diplay in the button
-	///   - style: The ButtonStyle, defaulting to primaryFilled
-	@objc public init(image: NSImage, style: ButtonStyle = .primaryFilled) {
-		super.init(frame: .zero)
-		imagePosition = .imageOnly
-		initialize(title: nil, image: image, style: style)
+	///   - size: The ButtonSize, default .large
+	///   - style: The ButtonStyle, default .primary
+	///   - accentColor: The accent or highlight NSColor, defaulting to the app primary color
+	@objc public convenience init(
+		image: NSImage,
+		size: ButtonSize = .large,
+		style: ButtonStyle = .primary,
+		accentColor: NSColor = Colors.primary
+	) {
+		let format = ButtonFormat(size: size, style: style, accentColor: accentColor)
+		self.init(image: image, format: format)
 	}
-	
-	
-	/// Initializes a Fluent UI Button with a title,  image,  imagePosition, and style
+
+	/// Initializes a Fluent UI Button with a title and image
 	/// - Parameters:
 	///   - title: String displayed in the button
-	///   - image: The NSImage to display in the button
-	///   - imagePosition: The position of the image, relative to the title
-	///   - style: The ButtonStyle, defaulting to primaryFilled
-	@objc public init(title: String, image: NSImage, imagePosition: NSControl.ImagePosition, style: ButtonStyle = .primaryFilled) {
-		super.init(frame: .zero)
-		self.imagePosition = imagePosition
-		initialize(title: title, image: image, style: style)
+	///   - image: The NSImage to diplay in the button
+	///   - imagePosition: The position of the image relative to the title, default .imageLeading
+	///   - size: The ButtonSize, default .large
+	///   - style: The ButtonStyle, default .primary
+	///   - accentColor: The accent or highlight NSColor, defaulting to the app primary color
+	@objc public convenience init(
+		title: String,
+		image: NSImage,
+		imagePosition: NSControl.ImagePosition = .imageLeading,
+		size: ButtonSize = .large,
+		style: ButtonStyle = .primary,
+		accentColor: NSColor = Colors.primary
+	) {
+		let format = ButtonFormat(size: size, style: style, accentColor: accentColor)
+		self.init(title: title, image: image, imagePosition: imagePosition, format: format)
 	}
 
-	/// Initializes a Fluent UI Button
-	/// Set style to primaryFilled as default
-	@objc public init() {
-		super.init(frame: .zero)
-		initialize(title: nil, image: nil, style: .primaryFilled)
+	/// Initializes a Fluent UI Button with no title or image, and with default formatting
+	@objc public convenience init() {
+		self.init()
```

Comment:
Why would someone want a button with no title or image?

Reply:
I did not add this initializer, I just made it better.  As for its purpose, someone might want to initialize the button (e.g. in an outer scope) and then add a title or image later.  That's pretty much what the designated NSButton initializer init(frame: .zero) does.

Reply:
I'm fairly sure we need the default initializer so we could use it within office where someone wanted to use the objective c keyword "new"